src/Components/Users/Users.tsx

```diff
@@ -19,15 +31,159 @@ export default function Users() {
     })
       .then((response) => {
         setUserList(response?.data?.data);
-        // setCurrentPage(pageNo);
       })
       .catch((error) => {
         console.log(error);
       })
-    }
+  }
+  //View Users API
+  const getUsersDetails = (id) => {
+    axios
+      .get(`${baseUrl}/Users/${id}`, {
+        headers: requestHeaders,
+      })
+
+      .then((response) => {
+        setUserDetails(response.data);
+
+      })
+      .catch((error) => {
+        console.log(error);
+      })
+  }
+  // toggle activated employee
+  const activatedEmployee = (itemId: number, data: any) => {
+    axios.put(`${baseUrl}/Users/${itemId}`, data, {
+      headers: requestHeaders,
+    })
+      .then((response) => {
+        console.log(response);
+
+        getAllUsers();
+        getToastValue("success", response?.data?.message || "Updated Sucessfully")
+      })
+      .catch((error) => {
+        console.log(error);
+
+        getToastValue("error", error?.response?.data?.message || "Error in updating the status");
+      })
+  }
+
+
+  // Toggle activation status
+  const toggleActivationStatus = (itemId: number, isActivated: boolean) => {
+    const updatedData = { isActivated: !isActivated };
+    activatedEmployee(itemId, updatedData);
+  };
+
+  //view Model
+  const showViewModel = (id: number) => {
+    setItemId(id);
+    setModelState("view-model")
+    getUsersDetails(id)
+  }
+
+  useEffect(() => {
+    getAllUsers()
+  }, [])
+
+  // Search by name
+  const getNameValue = (input) => {
+    console.log(input);
```

Comment:
consider removing logs before making a pull request, to avoid merging to the product branch (master) with logs in the user's browser, and these logs might reveal critical information.